packages/common/src/index.ts

```diff
@@ -1257,7 +1257,8 @@ export function formatValue<T>(format: string, value: T): string | T {
             if (Number.isNaN(value as any)) {
                 return value;
             }
-            return `${parseFloat(value as any) * 100}%`;
+            // Fix rounding issue
+            return `${(parseFloat(value as any) * 100).toFixed(2)}%`;
```

Comment:
@rephus do we not have access to the rounding here? whenever we have `format` I guess we have `round` too, we could fix to the correct number of places.

Reply:
not in the method, I guess I can pass that argument...

Reply:
actually, it is a bit hard, I'll have to change the call on the frontend and the backend and pass an optional argument just . 

And still it doesn't fix the issue if you don't specify a rounding, because this is a JS limitation..

Reply:
Is this a downside to having rounding occur on the backend? 

I say we merge this and reopen a new issue to move that out of the backend. I think it makes more sense to get the most raw data on the frontend, and then use rounding as a display (frontend) option. 

What do you think?

Reply:
mm, I'm not sure about this... I guess it makes more sense to do the formatting in the frontend, but then what was the point on having this `raw/formatted` response as we discussed ? 

I initially thought than doing Rounding at the DB level was more efficient, but we need to do it anyway for the format... and right now it doesn't seem to have a big impact (I'm not sure how many rows we normally support) 

We already passthrough the format flag from the schema, I guess we'll have to do the same for  `round` 

I guess I prefer to keep doing things at the backend, if we were not had to do the same operations on the frontend on echarts... so in this case I guess we should move everything to the frontend